app/tests/mock/test_mock_sentence_transformer.py

```diff
@@ -24,3 +27,28 @@ def dot_product(v1, v2):
     assert dot_product(long_text, long_text) > dot_product(long_text, medium_text)
     assert dot_product(long_text, medium_text) > dot_product(long_text, short_text)
     assert dot_product(medium_text, medium_text) > dot_product(medium_text, short_text)
+
+
+# This test doesn't normally run with the other tests because it requires downloading large embedding models.
+# To run this test, remove the `manual_` prefix so that the function begins with `test_`.
+# Run the test locally: pytest tests/mock/test_mock_sentence_transformer.py --capture=no -k test_sentence_transformer
+# The embedding models will be downloaded automatically to ~/.cache/huggingface/hub, if it does not already exist.
+@pytest.mark.parametrize(
+    "embedding_model", ["multi-qa-mpnet-base-cos-v1", "multi-qa-mpnet-base-dot-v1"]
+)
+def manual_test_sentence_transformer(embedding_model):
```

Comment:
Not following the intent here -- do we want to merge this code into the repo?

Reply:
Yes, I'd like to, in case we want to test other embedding models. Alternatively, I can move it to a regular file that is only used for testing.

Reply:
Can you share more about what this is intended to test?

Reply:
It's not a unit test so I'll move it into a separate file outside of `tests/`.
It exercises different embedding models and calculates scores from the embedding vectors. I used the scores to confirm/compare against those of pgvector's `max_inner_product`.